### PR TITLE
Improve Summoner CheckParams

### PR DESCRIPTION
**Pull request by Acbarakat**
There was an issue where Blood Pact Damage would from Pet: and Avatar: did not add up together. My solution is to make all Blood Pact Damage to be Pet: Blood Pact Damage.

### CODE REVIEW

File: addons/checkparam/checkparam.lua [hunk 1]
```diff
@@ -122,7 +122,13 @@ function get_text(id,data)
     local ext = extdata.decode({id=id,extdata=data})
     if ext.augments then
         for i,v in ipairs(ext.augments) do
-            split_text(id,v)
+			local stats = windower.regex.split(v,'(Pet|Avatar|Automaton|Wyvern|Luopan): ')
```
Comment (z16) [inline addons/checkparam/checkparam.lua]: Fix the indentation, from hard tabs to four spaces.

File: addons/checkparam/checkparam.lua [hunk 2]
```diff
@@ -122,7 +122,13 @@ function get_text(id,data)
...
+			if stats[2] then
+				local pet_text = windower.regex.replace(stats[2],'\n',' ')
```
Comment (z16) [inline addons/checkparam/checkparam.lua]: You should probably just use `stats[2]:gsub('\n', '')`, while the regex functions are very powerful they're usually somewhat slow if you don't need them. In this case you should also be able to use `stats[2]:trim()`.

**Review by z16 — CHANGES_REQUESTED** [review]

File: addons/checkparam/checkparam.lua (hunk 2 above)
Comment (Acbarakat) [reply addons/checkparam/checkparam.lua]: Changed and tested with gsub in the new commit

**Comment by z16** [comment]
I may have worded it poorly, but you need either `:gsub` or `:trim` but not both, sorry if I was unclear. `:trim()` removes any leading or trailing spaces (including newlines).

**Pull request merged**